guides/v2.2/comp-mgr/patching.md

```diff
@@ -28,7 +23,11 @@ Patch (or diff) files are text files that note:
 
 When the [patch][] program is run, this file is read in and the specified changes are made to the file(s).
 
-## Using the command line to apply a patch
+## Official patches
+
+You can find official Magento patches in the [Magento Security Center][]. Follow the instructions on the page to download the patch file, depending on your version and installation type.
+
+## Use the command line to apply a patch
```

Comment:
```suggestion
### Use the command line to apply an official patch
```

Reply:
I can't make a comment near line 36, but the patch command should be placed in a bash code block.

Reply:
@keharper, I think that's because you were looking at an outdated version of changes.